docs/pages/guides/using-eslint.mdx

```diff
@@ -3,33 +3,45 @@ title: Use ESLint and Prettier
 description: A guide on configuring ESLint and Prettier to format Expo apps.
 ---
 
-import { GithubIcon } from "@expo/styleguide-icons";
+import { GithubIcon } from '@expo/styleguide-icons';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { CODE } from '~/ui/components/Text';
 import { Step } from '~/ui/components/Step';
 import { Tabs, Tab, TabsGroup } from '~/ui/components/Tabs';
 import { Terminal } from '~/ui/components/Snippet';
+import { Collapsible } from '~/ui/components/Collapsible';
 
 <TabsGroup>
 
 [ESLint](https://eslint.org/) is a JavaScript linter that helps you find and fix errors in your code. It's a great tool to help you write better code and catch mistakes before they make it to production. In conjunction, you can use [Prettier](https://prettier.io/docs/en/), a code formatter that ensures all the code files follow a consistent styling.
 
 This guide provides steps to set up and configure ESLint and Prettier.
 
-## Setup
+## ESLint
+
+### Setup
+
+For SDK 51 and above, run the `npx expo lint` command which installs the `eslint` library and creates a **.eslintrc.js** config file at the root of your project.
+
+<Terminal cmd={['$ npx expo lint']} />
+
+The configuration in **.eslintrc.js** extends [`eslint-config-expo`](https://github.com/expo/expo/tree/main/packages/eslint-config-expo).
+
+<Collapsible summary="Setup instructions for SDK 50 and below">
 
 <Step label="1">
 
 Install ESLint, Prettier, and [`eslint-config-universe`](https://github.com/expo/expo/tree/main/packages/eslint-config-universe#eslint-config-universe) in your project.
```

Comment:
Is there any blocker for us to recommend `eslint-config-expo` package also for older SDK versions?

In theory, it's just a ESLint config, so we should be able to use or switch it no matter of Expo SDK version, since they should be unrelated. If we go that way it should be less complicated for the users, since there will be one recommendation no matter of SDK version user uses.

Reply:
The `npx expo lint` command doesn't work for SDK 50 and below. I'll verify about adding `eslint-config-expo` to an SDK 50 project and let you know.

Reply:
Tested this and it works and can confirm there are no blockers. Let me update this guide to add eslint-config-expo specific instructions for SDK 50 and below.

Thanks for catching this!

Reply:
@Simek, I've updated the instructions for SDK 50 and below to install `eslint-config-expo` instead of `eslint-config-universe`. The other changes still apply for SDK 50 because `npx expo lint` to lint project doesn't work.